client/components/pages/files/FilesTable.vue

```diff
@@ -3,23 +3,34 @@ import type {
   DataTablePageEvent,
   DataTableRowContextMenuEvent,
   DataTableRowDoubleClickEvent,
-  DataTableSortEvent,
+  DataTableSortMeta,
 } from 'primevue/datatable'
 import type { PContextMenu } from '#components'
 
 const DEFAULT_PAGE = 0
-const DEFAULT_PAGE_SIZE = 10
+const DEFAULT_PAGE_SIZE = 25
 
 const { locale, t, d } = useI18n()
+const filters = useFilesFilters()
+const { format } = useHumanFileSize()
 
 const selectedRows = defineModel<Array<FileRepresentationDTO>>('selectedRows', { default: () => [] })
 const selectedRow = ref<Nullable<FileRepresentationDTO>>(null)
+const sort = ref<DataTableSortMeta[]>([{ field: 'createdAt', order: -1 }])
+const query = defineModel<string>('query', { default: () => '' })
```

Comment:
```suggestion
const query = defineModel<string>('query', { default: '' })
```

we need a function for object types like objects and arrays, but not here

Reply:
Whoops, Copilot..